src/main/java/qupath/bioimageio/spec/BioimageIoSpec.java

```diff
@@ -1273,7 +1303,7 @@ public static class Binarize extends Processing {
 				super("binarize");
 			}
 
-			private double threshold = Double.NaN;
+			double threshold = Double.NaN;
```

Comment:
Why are these no longer private?

Reply:
Was just developing in a different file, then moved back - apologies. Although this one class file is getting rather large, so package-private might be needed again at some point